app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,349 @@
+    <string name="relationship_other">其他</string>
+    <string name="section_highest_rated_anime">评分最高的动漫</string>
+    <string name="section_most_popular_anime">最受欢迎的动漫</string>
+    <string name="section_top_upcoming_manga">热门即将推出的漫画</string>
+    <string name="section_highest_rated_manga">评分最高的漫画</string>
+    <string name="section_trending">本周趋势</string>
+    <string name="section_most_popular_manga">最受欢迎的漫画</string>
+    <string name="section_top_airing_anime">热播动漫</string>
+    <string name="preference_category_ui">用户界面</string>
+    <string name="preference_category_account">账户</string>
+    <string name="preference_category_advanced">高级</string>
+    <string name="preference_category_about">关于</string>
+    <string name="preference_dark_mode">深色模式</string>
+    <string name="preference_oled_black_mode">OLED 黑</string>
+    <string name="preference_oled_black_mode_description">在深色模式下使用黑色背景。</string>
+    <string name="preference_app_theme">主题</string>
+    <string name="preference_dynamic_color_theme">使用动态颜色</string>
+    <string name="preference_dynamic_color_theme_description">适应系统主题。</string>
+    <string name="preference_app_theme_default">默认</string>
+    <string name="preference_app_theme_green">绿</string>
+    <string name="preference_app_theme_purple">紫</string>
+    <string name="preference_app_theme_blue">蓝</string>
+    <string name="preference_media_item_size">海报图片尺寸</string>
+    <string name="preference_start_fragment">默认页面</string>
+    <string name="preference_language">语言</string>
+    <string name="preference_display_name">显示名称</string>
+    <string name="preference_profile_url">个人资料URL</string>
+    <string name="preference_profile_url_not_set">未设置个人资料 URL。</string>
+    <string name="preference_country">国家/地区</string>
+    <string name="preference_country_summary">国家/地区设置为%s</string>
+    <string name="preference_country_summary_non">未指定国家/地区。</string>
+    <string name="preference_titles">标题</string>
+    <string name="preference_adult_content">成人内容</string>
+    <string name="preference_adult_content_description_sfw">年龄分级为R18 +的媒体将被隐藏。</string>
+    <string name="preference_adult_content_description_everywhere">R18+年龄分级的媒体将随处可见。</string>
+    <string name="preference_rating_system">评分系统</string>
+    <string name="preference_remember_search_filters">记住搜索筛选条件</string>
+    <string name="preference_remember_search_filters_description">程序重启后使用上次使用过的搜索筛选规则。</string>
+    <string name="preference_force_legacy_image_picker">强制旧式照片选择器</string>
+    <string name="preference_force_legacy_image_picker_description">使用系统文件选择器，而不是新的照片选择器。</string>
+    <string name="preference_check_for_updates">检查发布时的更新</string>
+    <string name="preference_app_logs_description">查看并共享应用程序日志消息。</string>
+    <string name="preference_app_version">版本信息</string>
+    <string name="preference_github_repo">GitHub 代码库</string>
+    <string name="preference_open_source_libraries">\@string/nav_os_libraries</string>
```

Comment:
Could please remove these two strings `preference_open_source_libraries` and ` preference_app_logs`. I've marked them as not translatable in my latest commit.
```suggestion
```